python/tank/util/defaults_manager.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tank_vendor import shotgun_authentication as sg_auth
+
```

Comment:
I think there should be a clear comment in this file (maybe in the docstring for the mgr class) to point out that this is related to authentication. It's sort of subtle right now with the import up the top :)